### ISSUE TITLE
The problem is that my NVDA screen reader cannot read text in the terminal in VSCode (accessibility)

### ISSUE THREAD

**Issue by BKSLab** [microsoft/vscode#170144]
Type: <b>Bug</b>

Hello! The problem is that my NVDA screen reader cannot read text in the terminal in VSCode.
After I navigate to the terminal with the F6 key, I am prompted to use the arrows in combination with ctrl. But apart from the number of the list element, nothing is said. The same happens when moving with arrows without ctrl. You have to use the keys on the digital block, which is not very convenient for fluent reading. Tried changing terminal (Power Shell, Git Bash) but same result.
In the version of VSCode (1.70.2), I did not notice this behavior. The information in the terminal was read well, without additional effort. I then stayed on it.
However, now I need the latest version, and the question has become relevant again. Am I missing a setting or is this now considered normal behavior?

(VSCode - Version 1.74.2, NVDA - Version 2022.3.3, vocalizer_expressive2_driver-2022.1, Windows 10 Home Single Language: Version - 21H2, Build - 19044.2364)

VS Code version: Code 1.74.2 (e8a3071ea4344d9d48ef8a4df2c097372b0c5161, 2022-12-20T10:29:14.590Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz (8 x 1800)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|7.84GB (1.88GB free)|
|Process Argv|--crash-reporter-id 173f4d09-db5d-4b54-b9f4-10a0469ca2cb|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-sqlite|ale|0.14.1
vscode-language-pack-ru|MS-|1.74.12140931
flake8|ms-|2022.6.0
isort|ms-|2022.8.0
python|ms-|2022.20.1
vscode-pylance|ms-|2022.12.20
jupyter|ms-|2022.11.1003412109
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.12
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
vscccc:30610679
pyindex848cf:30577861
nodejswelcome1:30587005
282f8724:30602487
3d0df643:30613357
89544117:30613380
fim-prod:30623723

```

</details>

**Comment by joyceerhl** [comment]
Does this terminal accessibility documentation help? https://code.visualstudio.com/docs/editor/accessibility#_terminal-accessibility

**Comment by meganrogge** [comment]
I believe this is already fixed, can you see if you can reproduce in the [Insiders build](https://code.visualstudio.com/insiders)?
/label info-needed
